doc/source/conf.py

```diff
@@ -126,7 +126,7 @@ def check_sphinx_version(expected_version):
 
 # General information about the project.
 project = u'DrizzlePac'
-copyright = u'2017, Warren Hack, Nadia Dencheva, Chris Sontag, Megan Sosey, Michael Droettboom, Mihai Cara'
+copyright = u'2017, Warren Hack, Nadia Dencheva, Chris Sontag, Megan Sosey, Michael Droettboom, Mihai Cara, Mike Dulude, Michele De La Pena'
```

Comment:
Should the date also be updated to reflect the current version of the documentation?

Reply:
Updated to 2021.